src/platforms/unity/index.mdx

```diff
@@ -12,21 +17,99 @@ If you don't already have an account and Sentry project established, head over t
 
 ## Install
 
-Sentry captures data by using an SDK within your application’s runtime.
+Sentry captures data by using an SDK within your application’s runtime. 
+
+Install it from [`Unity UPM`](https://docs.unity3d.com/Manual/upm-ui.html) via [git url](https://docs.unity3d.com/Manual/upm-ui.html)
+
+```
+https://github.com/getsentry/sentry-unity.git?path=/package#{{ packages.version('sentry.dotnet.unity', '0.0.5') }}
```

Comment:
New install location: https://github.com/getsentry/unity

```suggestion
https://github.com/getsentry/unity.git#{{ packages.version('sentry.dotnet.unity', '0.0.5') }}
```

Good news is that the release process already updates the version:

![image](https://user-images.githubusercontent.com/1633368/114791948-6360c780-9d55-11eb-842c-cae6fb7c1a90.png)